src/main/java/com/meronat/iplog/IPLog.java

```diff
@@ -185,10 +192,21 @@ public Logger getLogger() {
 
     }
 
+    public PluginContainer getPluginContainer() {
```

Comment:
Do you plan to use this anywhere or is it just for convenience if we need it later?